scripts/check_license.sh

```diff
@@ -24,7 +24,7 @@ usage() {
     echo
     echo "Args:"
     echo "-i    Edit in-place."
-    echo "-v    Verbose, same as doing \`head -n 15\` on each file."
+    echo "-v    Verbose, same as doing \`head -n 16\` on each file."
```

Comment:
nit: this magical number should be automatically derived from the license file rather than hard coded.